app/views/stream_entries/_detailed_status.html.haml

```diff
@@ -32,7 +32,7 @@
   %div.detailed-status__meta
     %data.dt-published{ value: status.created_at.to_time.iso8601 }
     = link_to TagManager.instance.url_for(status), class: 'detailed-status__datetime u-url u-uid', target: stream_link_target, rel: 'noopener' do
-      %span= l(status.created_at)
+      %span= local_time(status.created_at, format: :default)
```

Comment:
Yeah, it uses `time.formats.default`, in this case, and both the ruby method and the JS support the bulk of the options available for localizing time formats.

Here's sample output:

    <span><time datetime="2017-04-16T16:38:48Z" data-local="time" data-format="%b %d, %Y, -- %H:%M">Apr 16, 2017, -- 16:38</time></span>

In this case the record was created at 16:38 UTC time, so that is what populates the datetime attribute and also what is in the value ... but then the JS converts this to local time (attached image)...

<img width="265" alt="screen shot 2017-04-20 at 9 47 25 pm" src="https://cloud.githubusercontent.com/assets/225/25259238/f8aab1de-2612-11e7-84bd-15921ebe0a31.png">

The formatting is preserved on the JS conversion.

I believe that the datetime formatting use here in the detailed status is the same as before.